### PR TITLE
feat: allow pro workspaces to use /upgrade to convert to ubb

### PR DESCRIPTION
**Pull request by alexnm**
image

To test:
* Ensure you have an old pro workspace (seat based, not flagged as ubb)
* Go to `/upgrade`, select the workspace
* Choose the pro plan, choose some addons, go over spending limit
* Last step is the accept changes now which should redirect to the portal instead of going to stripe

### CODE REVIEW

**Review by ed-codesandbox — APPROVED** [review]
I don't have a valid WS to check with at the moment - but the screenshot looks great 👌

File: packages/app/src/app/components/WorkspaceSetup/steps/Create.tsx [hunk 1]
```diff
@@ -70,6 +70,7 @@ export const Create: React.FC<StepProps> = ({
 
         setQueryParam('workspace', team.id);
         setFreshWorkspaceId(team.id);
+        await actions.dashboard.getTeams();
```
Comment (danilowoz) [inline packages/app/src/app/components/WorkspaceSetup/steps/Create.tsx:73]: Does this fix the issue with the sidebar, which couldn't find the current workspace?

File: packages/app/src/app/components/WorkspaceSetup/steps/ChangePlan.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+            <Text>{checkout.totalCredits} VM credits</Text>
+            <Text>{checkout.totalSandboxes} Sandboxes</Text>
+          </Stack>
+          <Text size={6} color="#e5e5e5">
+            ${checkout.totalPrice} / month
+          </Text>
+        </Stack>
+
+        <Text>
+          Additional VM credits are available on-demand for $0.018/credit.
```
Comment (danilowoz) [inline packages/app/src/app/components/WorkspaceSetup/steps/ChangePlan.tsx:65]: Isn't this value available in a constant?

**Review by danilowoz — APPROVED** [review]

File: packages/app/src/app/components/WorkspaceSetup/steps/ChangePlan.tsx (hunk 2 above)
Comment (alexnm) [reply packages/app/src/app/components/WorkspaceSetup/steps/ChangePlan.tsx:65]: Yes, could be a good idea as it's used in a few places

File: packages/app/src/app/components/WorkspaceSetup/steps/Create.tsx (hunk 1 above)
Comment (alexnm) [reply packages/app/src/app/components/WorkspaceSetup/steps/Create.tsx:73]: Yes, I think so. Weirdly on stream that problem was not reproducing, I'm curios what happens when this goes to prod

**Pull request merged**